src/main/java/org/infernalstudios/infernalexp/world/gen/features/LuminousFungusFeature.java

```diff
@@ -40,7 +40,7 @@ public boolean place(FeaturePlaceContext<NoneFeatureConfiguration> context) {
         int amount = 0;
 
         // Try to place luminous fungus 128 times
-        for (int i = 0; i < 128; i++) {
+        for (int i = 0; i < context.level().getHeight(); i++) {
```

Comment:
This should not be changed, as it is not related to the world height.

Reply:
My rationale for changing this number was that this is a feature that is placed from either the ceiling or floor and travels down or up respectively. This 128 marks the normal height of the nether so it is entirely possible for luminous fungus to make it from the bottom of the nether to the top. If datapacks change the nether height, it will no longer be able to do this, either being overly tall in smaller heights or too short with taller heights.

Reply:
It's an arbitrarily chosen number, because not every randomly generated position is a valid placement block. It has no bearing on how high it is, it's only the amount of tries the feature will try to place the fungus in the patch.